tools/cldr-code/src/main/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -381,4 +357,38 @@ private static ImmutableSet<String> buildPaths(String suffix) {
         }
         return builder.build();
     }
+
+    /**
+     * Return the PageId for the given emoji, making adjustments for pages that are united in
+     * emoji-test.txt but divided in Survey Tool, such as Symbols, Symbols2, and Symbols3
+     *
+     * @param emoji the emoji as a string
+     * @return the adjusted PageId
+     */
+    public static PageId getPageId(String emoji) {
+        final String major = getMajorCategory(emoji);
+        final String minor = getMinorCategory(emoji);
+        final PageId pageId = PageId.forString(major);
+        switch (pageId) {
+            case People:
+                if (minorToOrder.get(minor) < minorToOrder.get("person-fantasy")) {
+                    return PageId.People;
+                } else {
+                    return PageId.People2;
+                }
+            case Symbols:
+                if (minorToOrder.get(minor) < minorToOrder.get("transport-sign")) {
+                    if (pageId.getSectionId() != SectionId.Characters) {
+                        throw new RuntimeException("oops");
```

Comment:
I'll delete this after testing locally and before making ready for review

Reply:
Tested OK locally with Symbols, Symbols2, and Symbols3: there's no need to check the section id here so I'll remove that check

Reply:
Removed that check